PES2202300786 - Meghana Bisa

Meghanab1909 opened the pull request:
PES2202300786 - WEATHER APP - ASSIGNMENT 1

[WEATHER APP - ZIP.zip](https://github.com/SarthakSKumar/Next.js-13-PESU_IO-2023/files/12700392/WEATHER.APP.-.ZIP.zip)

Meghanab1909 committed:
Merge pull request #1 from SarthakSKumar/main

PES2202300786 - WEATHER APP - ASSIGNMENT 1

SarthakSKumar commented:
Hi. 
Please create a directory in Submissions/Assignment 1/ in the given format. ( [SRN]-Name ). Add your files under this directory. 
@Meghanab1909

SarthakSKumar reviewed (changes requested):
Please make the necessary changes

Meghanab1909 committed:
Add files via upload

Meghanab1909 committed:
Delete Submissions/Assignment 1/[SRN]-NAME/WEATHER APP - ZIP.zip

Meghanab1909 committed:
Add files via upload

Meghanab1909 committed:
Add files via upload

I have uploaded the HTML, CSS file along with images used in the html file. I have also uploaded 2 screenshots and a screen recording of the weather application.

Meghanab1909 committed:
Delete Submissions/Assignment 1/wind.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/Weather App - Asgmt 1.html

Meghanab1909 committed:
Delete Submissions/Assignment 1/Weather App - Screen Recording.mp4

Meghanab1909 committed:
Delete Submissions/Assignment 1/Weather App - Screenshot 1.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/Weather App - Screenshot 2.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/WeatherApp.css

Meghanab1909 committed:
Delete Submissions/Assignment 1/ash.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/tornado.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/clear.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/clouds.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/thunder.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/snow.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/smoke.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/drizzle.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/dust.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/humid.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/logo.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/mist.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/rain.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/sand.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/search.png

Meghanab1909 committed:
Delete Submissions/Assignment 1/[SRN]-NAME/PES2202300786 - MEGHANA BISA.zip

Meghanab1909 committed:
Create Weather App - Asgmt 1.html

Meghanab1909 committed:
Add files via upload

I have uploaded the HTML, CSS of the weather application along with the png images used in the HTML file. Along with the HTML, CSS and png files, 2 screenshots and a screen recording of the same has been uploaded

SarthakSKumar reviewed (approved):
Approved!
